packages/lib-classifier/src/plugins/tasks/SurveyTask/components/components/Chooser/components/CharacteristicsFilter/FilterStatus/FilterStatus.js

```diff
@@ -0,0 +1,83 @@
+import { Box, DropButton } from 'grommet'
+import { Filter } from 'grommet-icons'
+import PropTypes from 'prop-types'
+import React, { useRef, useState } from 'react'
+
+import Characteristics from '../Characteristics'
+import FilterButton from '../Characteristics/components/FilterButton'
+
+export default function FilterStatus (props) {
+  const { task } = props
+  const filterStatusRef = useRef()
+
+  // TODO: refactor filter state to model
+  const [ filters, setFilters ] = useState({})
+
+  function handleFilter (characteristicId, valueId) {
```

Comment:
removed `setTimeout` as [per PFE](https://github.com/zooniverse/Panoptes-Front-End/blob/master/app/classifier/tasks/survey/index.jsx#L62) because intended UI is changing - for PFE after click a filter [the modal should close](https://github.com/zooniverse/Panoptes-Front-End/pull/5076/files#r235142330), vs new UI modal with all filters should remain open while selecting filters and close when click away/Filter button.